frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -419,7 +439,12 @@ export default function AnalysisParamForm({
               </InputErrorContainer>
             </TwoParamContainer>
             <TwoParamContainer>
-              <Label htmlFor="widthFactorPeaks">Width (ms):</Label>
+              <Label
+                htmlFor="widthFactorPeaks"
+                title="Specifies how wide a signal should be to classify as a peak."
```

Comment:
I can't find a clear definition for peak width in scipy's docs, but signal reforms to the entire waveform so we probably want to use a different word. Maybe something like 
"Specifies the minimum required width of the base of a local max to be classified as a peak"